testsuite/classlibrary/TestDocument.sc

```diff
@@ -3,6 +3,41 @@ TestDocument : UnitTest {
 	// need a public place to communicate the result
 	classvar <>success;
 
+	test_save_method_writes_file {
+		var doc, file, str, path;
+		if(Platform.ideName == "scqt") {
+			path = Platform.defaultTempDir +/+ "test_document_save.txt";
```

Comment:
@muellmusik I think this technically should be an `.scd` document, but that's a really minor thing. Feel free to change if you like, otherwise we can merge as is.

Reply:
@muellmusik do you think this could be changed to `.scd` or do you want to leave it as is?

Reply:
What do you mean? You cannot declare classes in an scd file, so?

Reply:
Ah okay, sorry. Not sure why I did that tbh! Fixed.